pkgs/misc/vim-plugins/generated.nix

```diff
@@ -2472,14 +2472,14 @@ let
 
   neuron-vim = buildVimPluginFrom2Nix {
     pname = "neuron-vim";
-    version = "2020-08-07";
+    version = "2020-10-29";
     src = fetchFromGitHub {
-      owner = "ihsanturk";
+      owner = "fiatjaf";
       repo = "neuron.vim";
-      rev = "07521a3ef2940bd726e7b4d50b82e46898e686cc";
-      sha256 = "0myadiy6y2p73lhdzk2w55whg4i5rs004jaw1m21cz0dk8k8ibn2";
+      rev = "2b4321bf12a4d0b589cd8c26aabb7e31311dab26";
```

Comment:
Could you pick the latest rev (if it's not too much work)? There was a nice fix 2 days ago.

Reply:
https://github.com/fiatjaf/neuron.vim/commit/c62b5daa9bc9d734eca9db74e7e0895c85ceb2b8

Reply:
these automatically get updated through a script, the newest version should land in master shortly